### ISSUE TITLE
error reading

### ISSUE THREAD

**Issue by AlfredYang1986** [grafana/grafana#30211]
**Steps to reproduce**:

**Actual Result**:

**Expected Result**

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

**Environment**:
- Grafana version:
- Data source type & version:
- User OS & Browser:
- Others:

**Comment by grafanabot** [comment]
Thanks for creating this issue! We think it's missing some basic information. 

Follow the issue template and add additional information that will help us replicate the problem. 
For data visualization issues: 
- Query results from the inspect drawer (data tab & query inspector)
- Panel settings can be extracted in the panel inspect drawer JSON tab

For dashboard related issues: 
- Dashboard JSON can be found in the dashboard settings JSON model view

For authentication and alerting issues, Grafana server logs are useful. 

Happy graphing!
